### PR TITLE
docs: add canvas widget showcase

### PR DESCRIPTION
**Pull request by Valentin271**
Look dad!

### CODE REVIEW

**Comment by joshka** [comment]
> Look dad!

I'm sticking that on the fridge....

File: code/widget-showcase/canvas.tape [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# A VHS tape. See https://github.com/charmbracelet/vhs
+Output "../../src/content/docs/showcase/widgets/canvas.gif"
+Set Theme "Aardvark Blue"
+Set Width 600
+Set Height 400
```
Comment (joshka) [inline code/widget-showcase/canvas.tape]: Nit: On the website, this renders a bit narrower than the rest of the widgets.

**Review by joshka — COMMENTED** [review]
Amazing :D

File: code/widget-showcase/canvas.tape (hunk 1 above)
Comment (Valentin271) [reply code/widget-showcase/canvas.tape]: That's what I found was the most balanced in term of empty/drawn space.
But let me take another look see if I can have something better.

Comment (Valentin271) [reply code/widget-showcase/canvas.tape]: I found something that fits, pushed.

**Pull request merged**